frontend/app/(general)/settings/notes/page.tsx

```
@@ -10,6 +10,7 @@ import { getHighlightsByDroplet } from "@/lib/requests/highlights";
 import { PDFDocument } from "pdf-lib";
 import { NoteSummary } from "@/components/droplets/lessons/note-taking/note-summary";
 import { NotesManager } from "@/components/droplets/notes-manager";
+import { all } from "lowlight";
```

ESLint

[error] 'all' is defined but never used.

(@typescript-eslint/no-unused-vars)